app.py

```diff
@@ -41,13 +42,13 @@ def index():
         for name, value in request.form.items():
           if name.startswith("GENERATOR_"):
             variables[name] = value
-        uploaded_files = request.files.getlist("file[]")
-        filenames = []
+        uploaded_files = request.files.getlist("file")
         for file in uploaded_files:
             if file and allowed_file(file.filename):
                 filename = secure_filename(file.filename)
                 file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
-                filenames.append(filename)
+                os.rename(UPLOAD_FOLDER + filename, UPLOAD_FOLDER + 'files')
```

Comment:
What are you trying to do here?

Reply:
I was referring to this blog:- https://blog.fossasia.org/file-uploading-in-flask/
So after the `file.save` I just copied what was done in the blog

Reply:
I don't find any use of os.rename in the blog. @xeon-zolt could you add another review of the code. I am seriously confused.